src/components/createTrip/infoTrip/index.jsx

```diff
@@ -0,0 +1,193 @@
+import React from "react";
+import {Button, Tooltip, Collapse} from "antd";
+import moment from "moment";
+import {FiUser, FiCalendar} from "react-icons/fi";
+import {AiOutlineCar, AiOutlineArrowRight, AiOutlineInfoCircle} from "react-icons/ai";
+import {GiWeight} from "react-icons/gi";
+import {RiPinDistanceLine} from "react-icons/ri";
+import {IconContext} from "react-icons";
+import {DEFAULT_ICON_SIZE} from "../../../constants/icon";
+import {useState} from "react";
+import {useEffect} from "react";
+import {concatSettlements, concatThroughCities, manageTrip} from "../../../services/trips";
+import {tooltipMessages} from "../../../constants/tooltipeMessages/tripInfoMasseges";
+import {errorMessage} from "../../../services/alerts";
+import {tripsMessages} from "../../../constants/messages/trips";
+import {useHistory} from 'react-router-dom';
+
+const {Panel} = Collapse;
+
+function UserRoute(props) {
+    let history = useHistory();
+
+    const [allCities, setCities] = useState();
+    const [throughCities, setThroughCities] = useState();
+    const [dataManage, setDataManage] = useState();
+
+    useEffect(() => {
+        setCities(concatSettlements(props.props.points));
+        setThroughCities(concatThroughCities(props.props.points));
+        setDataManage({
+            tripId: props.props.id,
+            distance: props.distance,
+            totalWeight: props.props.loadCapacity
+        });
+    }, [props.distance])
+
+    let hasOffers = false;
+    if (props.creatTripData != null) {
+        hasOffers = props.creatTripData.pointsTrip.some(item => item.offerId != null);
+    }
+
+    if (props.totalWeigth >= props.props.loadCapacity) {
+        hasOffers = false;
+        errorMessage(tripsMessages.TITLE_OVERLOAD, tripsMessages.TEXT_OVERLOAD);
+    }
+
+    return (<IconContext.Provider value={{className: 'icon'}}>
+            {props.creatTripData != null ?
+                <div className="tripBody">
+                    <p className="dataField">
+                        <FiUser size={DEFAULT_ICON_SIZE}/>
+                        {props.props.fullName.name + " " + props.props.fullName.surname}
+                    </p>
+
+                    <div className="addresses">
+                        <p>
+                            From: {props.props.points[0].address + ", "
```

Comment:
Why don`t you use setPointAddress method?

Reply:
Because he is not needed here. What should I save the data to if it is stored in the component above.